security/monitoring.adoc

```diff
@@ -0,0 +1,197 @@
+[[security-monitoring]]
+= Monitoring cluster events and logs
+{product-author}
+{product-version]
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+toc::[]
+
+[[security-monitoring-intro]]
+== Introduction
+
+In addition to security measures mentioned in other sections of this
+guide, the ability to monitor and audit an {product-title} cluster is an
+important part of safeguarding the cluster itself and its users against
```

Comment:
s/the cluster itself and its/the cluster and its

Reply:
Fixed.